osler/inventory/views.py

```diff
@@ -118,3 +121,30 @@ def drug_dispense(request):
     else:
         drug.dispense(int(num))
     return redirect('inventory:drug-list')
+
+
+def export_csv(request):
```

Comment:
Can use @active_permission_required decorator here to enforce permissions. See export_pdf() function for example. Will also want to add context arguments to control whether the export button gets shown

Reply:
this branch doesn't have the new groups permissions code so I'm going to start a new one off master of move the export csv code over to that

Reply:
Yeah add the permission then test the permissions otherwise looks great